src/README.rst

```diff
@@ -16,9 +16,15 @@ To get started, after installation run the following:
 Change Log
 ==========
 
-Unreleased
+8.0.0
 ----------
-- Placeholder text
+- Improve help text on cluster select (#173)
```

Comment:
This is repeated from the last 7.1.0 release

Reply:
the readme for 8.0.0 has been removed to be in a follow up PR.